### DISCUSSION TITLE
I got the  problem about 'cv::imwrite_' when I run engine.test() in 001_getting_started.ipynb

### DISCUSSION THREAD

**Discussion by hanselliu** [open-edge-platform/anomalib#1867] [Q&A] [answered]
I just followed the notebook step bay step but still got this problem. Could someone kindly help me?
![image](https://github.com/openvinotoolkit/anomalib/assets/96777424/e70ac446-ceb6-4271-8db1-eb1a64228f06)

**Comment by blaz-r** [comment, marked as answer]
Hi, this was discussed in #1802 and I believe the solution was introduced here: #1831 potentially to be merged by #1866 .

**Reply by hanselliu** [reply]
Thanks a lot! It did work after I changed anomalib.data.utils.image.py:444.
